Entities/CustomRespawnTimeRefill.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Monocle;
+using MonoMod.Utils;
+using System;
+
+namespace Celeste.Mod.SpringCollab2020.Entities {
+    [CustomEntity("SpringCollab2020/CustomRespawnTimeRefill")]
+    class CustomRespawnTimeRefill : Refill {
+        public CustomRespawnTimeRefill(EntityData data, Vector2 offset) : base(data, offset) {
+            DynData<Refill> self = new DynData<Refill>(this);
+            float respawnTime = data.Float("respawnTime", 2.5f);
+
+            foreach (Component component in this) {
+                if (component is PlayerCollider collider) {

```

Comment:
From what I can see, there is only a single `PlayerCollider` attached to the `Refill` at all times. `Get<PlayerCollider>()` should work just fine.

Reply:
Definitely, I just didn't see this method until now. 😅